internal/modules/workloads/detail_describer.go

```diff
@@ -124,7 +125,12 @@ _%s_
 		objects = append(objects, &pods[i])
 	}
 
-	rv, err := resourceviewer.Create(ctx, options.Dash, options.Queryer, objects...)
+	sort.Slice(objects, func(i, j int) bool {
+		return objects[i].GetName() < objects[j].GetName()
```

Comment:
I don't really understand why sorting by name gives you the initial selection?

Reply:
It's important to sort pods, otherwise in consecutive updates selection will go from one pod to another, resulting in screen flicker.